src/Illuminate/Auth/Access/HandlesAuthorization.php

```diff
@@ -50,6 +50,8 @@ public function denyWithStatus($status, $message = null, $code = null)
      */
     public function denyAsNotFound($message = null, $code = null)
     {
+        $message = $message ?? '404 not found.';
```

Comment:
I think the emssage should simply be "`Not found.`" (mind the uppercase letter and the final period).

As it is used here:

https://github.com/laravel/framework/blob/2a1a55caf3c65a74f173f47be594587cb931f22c/src/Illuminate/Foundation/Exceptions/Handler.php#L385

No other exception message thrown in the framework adds the status code to its message text.

Reply:
You are right! I will change that.